server/database/dummy_data/weekly_event.js

```diff
@@ -0,0 +1,27 @@
+const WeeklyEvents = require("../models/WeeklyEvent");
+const Mood = require("../models/Mood")
+const User = require("../models/User")
+
+const buildWeeklyEvents = async () => {
+  // get moods 
+  const moods = await WeeklyEvents.find();
```

Comment:
should it be WeeklyEvents.find()?
```suggestion
  const moods = await Mood.find();
```